blaze/compute/tests/test_mongo_compute.py

```diff
@@ -8,12 +8,19 @@
 from toolz import pluck, reduceby, groupby
 
 from datashape import Record
-from blaze import into, compute, compute_up, discover, dshape, data
+from blaze import into as bz_into, compute, compute_up, discover, dshape, data
```

Comment:
Agreed in general; in this case we redefine and wrap `blaze.into` to detect Mongo connection failures, defined below.